apps/vss_generation_timetable_mip_testing.cpp

```diff
@@ -0,0 +1,65 @@
+#include "solver/mip-based/VSSGenTimetableSolver.hpp"
+
+#include <gsl/span>
+
+int main(int argc, char** argv) {
+  if (argc != 11) {
+    std::cout << "Expected 10 arguments, got " << argc - 1 << std::endl;
+    std::exit(-1);
+  }
+
+  auto args = gsl::span<char*>(argv, argc);
```

Comment:
Including an entire library only for this seems a bit overkill to me. You can do what you did before and just silence the Linter complaint, imo.

Reply:
True, this is due to a linter message and its suggestion. I would leave it for now, we might enhance the user input in the future anyways and it is sensible to finalize this part then.